tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Steps/OrganisationServiceRecipientSteps.cs

```diff
@@ -54,7 +54,11 @@ public async Task ThenTheOrganisationServiceRecipientIsReturnedWithTheFollowingV
 
         private static object CreateServiceRecipients(JToken token)
```

Comment:
The return type can be `ServiceRecipientsTable` as well now.

Reply:
👍